app/views/users/new.html.haml

```diff
@@ -0,0 +1,13 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user, html: {class: "form_horizontal"} do |f|
```

Comment:
Read the bootstrap form for docs. There is a better way how to deal with form_horizontal class.

HOMEWORK: read bootstrap form for docs and apply new knowledge.